src/fde.h

```diff
@@ -97,6 +114,18 @@ class fde
     /// record a transaction on this FD
     void noteUse() { ++pconn.uses; }
 
+    /// record bytes being read and return the accumulated total for this FD.
+    void bytesRead(const size_t n) { bytes_read += n; }
+
+    /// \sa bytes_read
```

Comment:
Just to minimize the number of links documentation readers have to click to learn what this method does:

```suggestion
    /// \copydoc totalBytesRead_
```

The specific suggestion above accounts for the renaming suggestion in the other change request. Adjust as needed.